src/core/zowe/core_for_zowe_sdk/session.py

```diff
@@ -79,7 +82,7 @@ def __init__(self, props: dict) -> None:
         self.session.basePath = props.get("basePath")
         self.session.port = props.get("port", self.session.port)
         self.session.protocol = props.get("protocol", self.session.protocol)
-        self.session.rejectUnauthorized = props.get("rejectUnauthorized", self.session.rejectUnauthorized)
+        self.session.rejectUnauthorized = False if props.get("rejectUnauthorized") == "False" else True
```

Comment:
Same as previous comment, but feel free to only update these in the Auth_Type branch 😋